src/routes/Routes.tsx

```diff
@@ -69,210 +67,185 @@ const MSG = defineMessages({
 });
 
 const Routes = () => {
-  const { user, wallet } = useAppContext();
+  const { user } = useAppContext(); // wallet
   const isMobile = useMobile();
   // const isAppLoading = appLoadingContext.getIsLoading();
-
-  // disabling rules to silence eslint warnings
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const isConnected = wallet?.address;
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const didClaimProfile = user?.name;
+  // const isConnected = wallet?.address;
+  // const didClaimProfile = user?.name;
 
   useTitle();
 
-  /**
-   * @NOTE Memoized Switch
-   *
-   * We need to memoize the entire route switch to prevent re-renders at not
-   * so oportune times.
-   *
-   * The `balance` value, accessible through (no longer isLoggedInUser), even if we don't
-   * use it here directly, will cause a re-render of the `<Routes />` component
-   * every time it changes (using the subscription).
-   *
-   * To prevent this, we memoize the whole routes logic, to only render it again
-   * when the user connects a new wallet.
-   *
-   * This was particularly problematic when creating a new colony, and after
-   * the first TX, the balance would change and as a result everything would
-   * re-render, reseting the wizard.
-   */
-  const MemoizedSwitch = useMemo(
-    () => (
-      <RoutesSwitch>
-        <Route path="/" element={<Navigate to={LANDING_PAGE_ROUTE} />} />
-        <Route path={NOT_FOUND_ROUTE} element={<FourOFour />} />
-        <Route
-          path={LANDING_PAGE_ROUTE}
-          element={
+  // if (isAppLoading) {
+  //   return <LoadingTemplate loadingText={MSG.loadingAppMessage} />;
+  // }
```

Comment:
Is this still relevant? It might be worth creating an issue for it or removing the comment from code

Reply:
Good catch! I think this code is really old and no longer relevant